programs/cleos/CMakeLists.txt

```diff
@@ -13,6 +15,11 @@ target_include_directories(${CLI_CLIENT_EXECUTABLE_NAME} PUBLIC ${CMAKE_CURRENT_
 target_link_libraries( ${CLI_CLIENT_EXECUTABLE_NAME}
                        PRIVATE appbase version leap-cli11 chain_api_plugin producer_plugin chain_plugin http_plugin eosio_chain fc ${CMAKE_DL_LIBS} ${PLATFORM_SPECIFIC_LIBS} )
 
+if (CURL_FOUND)
```

Comment:
don't need the `if (CURL_FOUND)` now that curl is `REQUIRED` above

Reply:
I intentionally keep it that way to make it more clear what needs to changed whenever other library other than libcurl is used. You described that intention in the issue. I can certainly remove it if you insisted.

Reply:
It's not critical, you can leave it as is if you'd like, because yes it will be changed when I get around to proper macOS support here.